content/docs/setup/index.md

```diff
@@ -8,7 +8,8 @@ Description: 'Get started with maticjs'
 
 In order to support multiple web3 libraries - maticjs provides plugins.
 
-Currently two libraries are supported -
+Currently three libraries are supported -
 
 1. Web3.js
 2. Ethers
+3. Unstoppable Domains
```

Comment:
`Unstoppable Domains` is not web3 library, so its better to add in seperate context.

You can add a line below saying - `You can use unstoppable domains plugin for Polygon name spacing` or something equivalent.

Reply:
Thank you for highlighting this, I just made a push that fixes this